### DISCUSSION TITLE
webpushtest.com error on iPhone SE 2nd gen, iOS 16.6.1

### DISCUSSION THREAD

**Discussion by Leftium** [magicbell/community#137] [General]
(It works on iPhone 13 mini.)

However when I try webpushtest.com on iPhone SE 2nd gen, iOS 16.6.1:

![image](https://github.com/magicbell-io/public/assets/381217/286e57fc-46d9-4d6f-8014-89998eaa4aa4)

```
{
  "standalone": true,
  "browserName": "Mobile Safari",
  "osName": "iOS",
  "deviceType": "mobile",
  "osVersion": "16.6.1",
  "isPrivate": false,
  "notificationApiPermissionStatus": "default",
  "serviceWorkerStatus": "fetching",
  "subscriptionState": "pending",
  "topics": []
}
```

**Comment by unamashana** [comment]
Hey @Leftium have you allowed Safari to send push notifications in the iOS settings? Everything else seems good to me in the screenshot you posted. By the way iOS 17 is out now, and might be worth upgrading just to be sure - even when iOS 16.6 should technically support the feature.

**Reply by Leftium** [reply]
I did not know `Push Safari > Advanced > Experimental Features > Push API` had to be [manually enabled](https://www.xda-developers.com/how-enable-safari-notifications-iphone/). iPhone 13 mini (iOS 17) must have this enabled by default.

However, I still get the same error on iPhone SE (iOS 16.6) even after enabling `Push API`. I tried deleting the first bookmark on the home screen, reloading webpushtest.com, and newly adding to the home screen again.

I could not find "the website's notifications settings" as described and depicted in the link above, even on the iPhone 13 mini, where webpushtest.com simply worked.

**Reply by Leftium** [reply]
Oh, "the website's notifications settings" is specific to twitter.com. It must be equivalent to hitting the `Subscribe` button on webpushtest.com.

I tried enabling web notifications for twitter.com on iPhone SE iPhone SE (iOS 16.6), and it seems to have worked. So I think there is a problem with webpushtest.com.

**Reply by unamashana** [reply]
So does it mean WebPushTest.com works for you on iOS17? I am not quite sure how to help, because we are unable to test on 16.6 ourselves. I am sorry about that.

**Reply by Leftium** [reply]
@unamashana WebPushTest.com fails on iPhone SE2 regardless of iOS version.

But notifications from Twitter PWA work on iPhone SE2. So this leads me to believe there is a problem with WebPushTest.com.
